31.md

```diff
@@ -0,0 +1,28 @@
+NIP-31
+======
+
+Pinned Events
+-------------
+
+`draft` `optional` `author:plantimals`
+
+NIP-31 amends the content metadata for `Kind 0` as described in [NIP-01](01.md).
+
+Each pubkey MAY indicate a pinned event by setting the field `"pinned_event"`
```

Comment:
This should use `e` tags instead with a `pinned` marker, otherwise only 1 pinned event is allowed. Clients can choose to display 1 or more.
e.g.
```
{
  ...
  "tags": [["e","<eventId>", "<relayUrl>", "pinned"]]
  ...
}
```